src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/AddStandardAddressCheckRequestWrapper.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.entity
+
+import jakarta.validation.Valid
+import uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.model.AddStandardAddressCheckRequest
+
+data class AddStandardAddressCheckRequestWrapper(
+  @Valid val addStandardAddressCheckRequest: AddStandardAddressCheckRequest,
+  @Valid val agent: Agent,
```

Comment:
Could we have an abstract class that AddStandardAddressCheckRequest, AddResidentsRequestWrapper, AddCasCheckRequest inherit, something like AbstractAgentRequest?